rest/handler.go

```diff
@@ -201,6 +203,17 @@ func (h *handler) invoke(method handlerMethod) error {
 		}
 	}
 
+	// If an Admin Request we need to check the user credentials
+	// FIXME: Temporarily disable this
+	if false {
+		clusterAddress, _, _, _, _, _ := tempConnectionDetailsForManagementEndpoints()
```

Comment:
Why? Is it because we don't want to break usages of the Admin API? At what point is it safe to enable this permanently?

Reply:
Two reasons:
- Don't want to break usages of admin API at the moment
- Haven't yet defined the list of permissions for each endpoint

I'm confident the testing shows this works so when I go back and add permissions for each endpoint I can enable this along with (probably) some default for tests setting the override config

Reply:
I'd also prefer not to have 'if false' in this way if it's going into master and not a dev branch.  Even having a 'check permissions' function that's stubbed out and always returns true would be preferred I think - at least then it's clear how this code is expected to interact with the permissions set.

Reply:
Okay moved the 'if' work into `shouldCheckAdminAuth` for now.